src/SFML/Window/Unix/WindowImplX11.cpp

```diff
@@ -77,14 +77,14 @@ namespace
 
         sf::String                            wmAbsPosGood[] = { "Enlightenment", "FVWM", "i3" };
 
-        static const unsigned long            eventMask = FocusChangeMask      | ButtonPressMask     |
+        static constexpr unsigned long        eventMask = FocusChangeMask      | ButtonPressMask     |
                                                         ButtonReleaseMask    | ButtonMotionMask    |
                                                         PointerMotionMask    | KeyPressMask        |
                                                         KeyReleaseMask       | StructureNotifyMask |
                                                         EnterWindowMask      | LeaveWindowMask     |
                                                         VisibilityChangeMask | PropertyChangeMask;
```

Comment:
Should the following lines be indented as well?

Reply:
That would probably be undone by https://github.com/SFML/SFML/pull/1861, so not sure if it's worth to invest any effort in horizontal formatting.